src/components/movies/filter/filter.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { Item, LeftFilter, RightSort, TextSort, Wrapper } from './filter-styles';
+
+const filter = ["all", "documentary", "comedy", "horror", "crime"];
+const sort = ['Date', 'Genre', 'Title'];
+
+const Filter = () => (
+  <Wrapper>
+    <LeftFilter>
```

Comment:
Wrong names for filters and sort components, they should be without `left` and `right` prefixes.
If you will need to put them in other place or in other order, you will need also to rename this components, which is not good
They should be called `FiltersSection` and `SortSection`, smth like this